src/main/java/com/github/onsdigital/dp/uploadservice/api/APIClient.java

```diff
@@ -1,4 +1,132 @@
 package com.github.onsdigital.dp.uploadservice.api;
-public class APIClient {
 
-}
\ No newline at end of file
+import com.github.onsdigital.dp.uploadservice.api.exceptions.ConnectionException;
+import org.apache.hc.client5.http.classic.methods.HttpPost;
+import org.apache.hc.client5.http.entity.mime.MultipartEntityBuilder;
+import org.apache.hc.client5.http.impl.classic.CloseableHttpClient;
+import org.apache.hc.client5.http.impl.classic.HttpClients;
+import org.apache.hc.core5.http.ContentType;
+import org.apache.hc.core5.http.HttpEntity;
+import org.apache.hc.core5.http.NameValuePair;
+import org.apache.hc.core5.http.io.HttpClientResponseHandler;
+import org.apache.hc.core5.http.io.entity.EntityUtils;
+import org.apache.hc.core5.http.message.BasicNameValuePair;
+import org.apache.hc.core5.net.URIBuilder;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.util.List;
+
+import static com.github.onsdigital.logging.v2.event.SimpleEvent.info;
+
+public class APIClient implements Client  {
+
+    private static final int CHUNK_SIZE = 1024 * 1024 * 5; // 5MB chunks
```

Comment:
This should be set from a config value so it can be easily amended in future

Reply:
This is now addressed.